src/drivers/driver_zephyr.c

```diff
@@ -1430,6 +1449,42 @@ static int wpa_drv_zep_get_conn_info(void *priv, struct wpa_conn_info *ci)
 	return ret;
 }
 
+#ifdef CONFIG_AP
+static int wpa_drv_zep_set_ap(void *priv,
+			      struct wpa_driver_ap_params *params)
+{
+	struct zep_drv_if_ctx *if_ctx = NULL;
+	const struct zep_wpa_supp_dev_ops *dev_ops = NULL;
+	int ret = -1;
+
+	if ((!priv) || (!params)) {
+		wpa_printf(MSG_ERROR, "%s: Invalid params\n", __func__);
+		goto out;
+	}
+
+	if_ctx = priv;
+
+	dev_ops = if_ctx->dev_ctx->config;
+
+	if (!dev_ops->start_ap) {
+		wpa_printf(MSG_ERROR, "%s: set_ap op not supported\n", __func__);
+		goto out;
+	}
+
+	ret = dev_ops->start_ap(if_ctx->dev_priv,
```

Comment:
Can we have some comment describing why `set_ap` needs to call `start_ap` ?

Reply:
Rejigged the ops.